<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbViewDialog</class>
 <widget class="QDialog" name="DbViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据库视图</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>901</width>
     <height>760</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableView" name="userTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>830</width>
       <height>660</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableView" name="prodTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>830</width>
       <height>660</height>
      </rect>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>40</number>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QToolButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>750</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>750</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>670</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
